### PR TITLE
#4020 This looks like a typo in the description?

### PR DESCRIPTION
**Pull request by dietmarw**
https://github.com/modelica/ModelicaStandardLibrary/issues/4020

### CODE REVIEW

**Review by maltelenz — APPROVED** [review]
Looks good to me.

File: Modelica/Electrical/Analog/Examples/ResonanceCircuits.mo [hunk 1]
```diff
@@ -6,7 +6,7 @@ model ResonanceCircuits
   parameter SI.Capacitance C=0.01 "Capacitance";
   parameter SI.Inductance L=0.01 "Inductance";
   final parameter SI.Frequency fRes=1/(2*pi*sqrt(L*C)) "Source frequency";
```
Comment (HansOlsson) [inline Modelica/Electrical/Analog/Examples/ResonanceCircuits.mo:8]:
```suggestion
  final parameter SI.Frequency fRes=1/(2*pi*sqrt(L*C)) "Resonance frequency";
```
I agree that the next line should say "Source to resonance frequency", but that doesn't make sense in itself if both of the frequencies are called 'Source frequency'.

**Review by HansOlsson — CHANGES_REQUESTED** [review]
I believe this makes more sense combined with the suggestion.

**Comment by dietmarw** [comment]
I agree @HansOlsson. Already updated.

**Review by HansOlsson — APPROVED** [review]

**Pull request merged**